packages/presets/src/fragments/copilot-panel/copilot-service/copilot-config.ts

```diff
@@ -24,6 +24,43 @@ type VendorPack<T extends AllServiceKind> = {
   impl: ServiceImpl<GetMethod<T>, unknown>;
 };
 
+class SimplePRNG {
+  constructor(private seed: number) {}
+  // 一个简单的PRNG，使用线性同余生成器
```

Comment:
Please mind not using Chinese comment, and link to some reference material of this kind of algorithm would be better